scripts/Muon/GUI/FrequencyDomainAnalysis/MaxEnt/maxent_presenter_new.py

```diff
@@ -10,27 +10,48 @@
 import math
 from Muon.GUI.Common import thread_model
 import mantid.simpleapi as mantid
+from Muon.GUI.Common.utilities.algorithm_utils import run_MuonMaxent
+import re
+from Muon.GUI.Common.observer_pattern import Observer
+from Muon.GUI.Common.thread_model_wrapper import ThreadModelWrapper
+import functools
+from Muon.GUI.Common.ADSHandler.workspace_naming import get_maxent_workspace_group_name, get_maxent_workspace_name, get_base_data_directory
+from Muon.GUI.Common.ADSHandler.muon_workspace_wrapper import MuonWorkspaceWrapper
+
 
 raw_data = "_raw_data"
 
 
+class GenericObserver(Observer):
```

Comment:
should this be in https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/Common/observer_pattern.py?